sdk/python/lib/pulumi/__init__.py

```diff
@@ -128,6 +131,8 @@
     "deprecated",
     # errors
     "RunError",
+    "InputPropertiesError",
+    "InputPropertyError",
```

Comment:
I think we also want to "export" `PropertyError` here, or whatever we end up calling it.

Reply:
:heavy_check_mark: